### ISSUE TITLE
Exporting a PDF does not work with vite

### ISSUE THREAD

**Issue by danhooper** [amcharts/amcharts5#833]
**Bug description**

When exporting a PDF using vite the export throws an exception. 

I created a reproducible example at https://github.com/danhooper/amcharts-vite-bug

Checkout the repo and run:
```
npm i
npm run dev
```

When clicking the export and PDF option you will see an error logged to the console.

```
Uncaught (in promise) TypeError: "vfs" is read-only
    _pdfmake Exporting.ts:52
    fulfilled tslib.es6.js:118
    promise callback*step tslib.es6.js:120
    __awaiter tslib.es6.js:121
    __awaiter tslib.es6.js:117
    _pdfmake @amcharts_amcharts5_plugins_exporting.js:36
    getPdfmake Exporting.ts:1749
    getPDF Exporting.ts:1507
    fulfilled tslib.es6.js:118
    promise callback*step tslib.es6.js:120
    __awaiter tslib.es6.js:121
    __awaiter tslib.es6.js:117
    getPDF @amcharts_amcharts5_plugins_exporting.js:729
    exportPDF Exporting.ts:1474
    __awaiter tslib.es6.js:121
    __awaiter tslib.es6.js:117
    exportPDF @amcharts_amcharts5_plugins_exporting.js:706
    export Exporting.ts:771
    __awaiter tslib.es6.js:121
    __awaiter tslib.es6.js:117
    export @amcharts_amcharts5_plugins_exporting.js:145
    download Exporting.ts:712
    __awaiter tslib.es6.js:121
    __awaiter tslib.es6.js:117
    download @amcharts_amcharts5_plugins_exporting.js:104
    _handleClick ExportingMenu.ts:489
    createItems ExportingMenu.ts:461
    addEventListener Utils.ts:63
    createItems ExportingMenu.ts:460
    each Array.ts:93
    createItems ExportingMenu.ts:422
    _afterChanged ExportingMenu.ts:353
    _runDirties Root.ts:879
    _runDirties Root.ts:878
    _renderFrame Root.ts:897
    _runTicker Root.ts:922
    _ticker Root.ts:957
    rAF Root.ts:44
    _startTicker Root.ts:960
    _addDirtyEntity Root.ts:986
    markDirty Entity.ts:1170
    _markDirtyKey Entity.ts:485
    _markDirtyPrivateKey Entity.ts:1103
    _setPrivate Entity.ts:753
    setPrivate Entity.ts:770
    setPrivate Entity.ts:1267
    _setDefaults Language.ts:299
    _afterNew Entity.ts:1013
    new Entity.ts:980
    Root Root.ts:171
    new Root.ts:377
    setupChart chart.ts:11
    <anonymous> main.ts:10
```

**Environment (if applicable)**

 - amCharts version: 5.3.6
 - Browser and its version: Firefox 110 (also verified on Chrome 110)
 - Related frameworks and their versions: vite, typescript

**Additional context**

It looks like dynamic imports cannot be written to which is what is being done in the Exporting.ts file

```
/**
 * Loads pdfmake dynamic module
 *
 * This is an asynchronous function. Check the description of `getImage()`
 * for description and example usage.
 *
 * @ignore
 */
async function _pdfmake(): Promise<any> {
	let a = await Promise.all([
		import(/* webpackChunkName: "pdfmake" */ "pdfmake/build/pdfmake.js"),
		import(/* webpackChunkName: "pdfmake" */ "./pdfmake/vfs_fonts")
	]);
	let pdfmake = a[0];
	let vfs_fonts = a[1];
	const global = <any>window;
	global.pdfMake = global.pdfMake || {};
	global.pdfMake.vfs = vfs_fonts.default;
	pdfmake.vfs = vfs_fonts.default;
	return pdfmake;
}
```

**Comment by martynasma** [comment]
Thanks for the report. We're working on ways to solve this.

Will keep you posted.

**Comment by Pauan** [comment]
Thanks for the detailed report! This should be fixed in the next version.

**Comment by martynasma** [comment]
Fixed in [5.3.7](https://github.com/amcharts/amcharts5/releases/tag/5.3.7).

## [5.3.7] - 2023-03-09

### Added
- New `calculateSize` setting on `Root`. This is needed if you are using a `transform: scale(...)` CSS style. [More info](https://www.amcharts.com/docs/v5/getting-started/root-element/#Custom_sizing_function).
- New settings on `DateRangeSelector`: `minDate` and `maxDate`. Allows restricting date selection to specific date ranges. [More info](https://www.amcharts.com/docs/v5/charts/stock/toolbar/date-range-selector/#Restricting_date_range).
- New events on `StockChart`: `drawingsupdated` and `indicatorsupdated`. Events kick in when drawings or indicators are added, updated, or removed. [More info](https://www.amcharts.com/docs/v5/charts/stock/serializing-indicators-annotations/#Events).

### Changed
- `DateRangeSelector` will now limit dates to actual range of data. To disable set `minDate` and `maxDate` to `null`. [More info](https://www.amcharts.com/docs/v5/charts/stock/toolbar/date-range-selector/#Restricting_date_range).

### Fixed
- `XYSeries` were creating `Bullet` objects even if distance between them was less than `minBulletDistance` which was not efficient and could slow the browser down.
- Fixed issue with mutating ES6 modules ([Issue 833](https://github.com/amcharts/amcharts5/issues/833)).
- Fixed more issues with `ghostLabel` obstructing interactivity for other labels.
- Bullet function was not being called when a single data item of the series was updated.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
